content/about/index.nl.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Wie zijn wij?
+menu: main
+---
+
+Sol Nascente is een non-profitorganisatie die ernaar streeft het noorden van Portugal en Spanje te regenereren en te herbebossen door te bewijzen dat een bos zowel ecologisch gezond als enorm productief kan zijn.
+
+#### Missie
+Het is onze missie is om aangetaste gronden aan te pakken en te transformeren tot bloeiende bosgemeenschappen. Om land te verwerven dat verzwakt is door generaties van verwaarlozing en het te beheren met regeneratieve agrobosbouw om de bodem te herstellen en bosbranden te weerstaan. Het biodivers bos dat ontstaat levert een verscheidenheid aan oogst die we vervolgens verwerken tot producten met een hoge toegevoegde waarde. Een bos met een nieuwe manier van bosbeleving door natuurlijke schoonheid, opbouw van koolstofkredieten, met behoud van biodiversiteit, dat ecosysteemdiensten levert gebaseerd op de bosmodellen die ze met elkaar verbinden.
```

Comment:
Deze eerste zin is echt raar geformuleerd. Zou je hier nog even naar kunnen kijken, voordat ik het online gooi?